src/VirtoCommerce.InventoryModule.Data/Services/FulfillmentCenterSearchService.cs

```diff
@@ -33,26 +33,25 @@ public virtual async Task<FulfillmentCenterSearchResult> SearchCentersAsync(Fulf
             var cacheKey = CacheKey.With(GetType(), nameof(SearchCentersAsync), criteria.GetCacheKey());
             return await _platformMemoryCache.GetOrCreateExclusiveAsync(cacheKey, async (cacheEntry) =>
             {
+                using var repository = _repositoryFactory();
```

Comment:
Are there any significant changes except formatting and new `using` here? 
Maybe we could revert this file?

Reply:
It's just SLint warning.

Reply:
I agree with @yecli  we need to stick to our current `using` code style.

Reply:
Agree, old using style seems more clear. Maybe we could disable that Slint warning.